docs/contributing.rst

```diff
@@ -124,14 +124,15 @@ Example: ``./test/libsolidity/syntaxTests/double_stateVariable_declaration.sol``
     // ----
     // DeclarationError: (36-52): Identifier already declared.
 
-A syntax test must contain at least the contract under test itself, followed by the separator ``// ----``. The following comments are used to describe the
-expected compiler errors or warnings. The number range denotes the location in the source where the error occurred.
-In case the contract should compile without any errors or warning, the section after the separator has to be empty
-and the separator can be left out completely.
+A syntax test must contain at least the contract under test itself, followed by the separator ``// ----``.
+You use the comments to describe the expected compiler errors or warnings. The number range denotes the
+location in the source where the error occurred.
+If you want the contract to compile without any errors or warning, leave the section after the separator empty
+and you can leave out the separator completely.
 
-In the above example, the state variable ``variable`` was declared twice, which is not allowed. This will result in a ``DeclarationError`` stating that the identifier was already declared.
+In the above example, the state variable ``variable`` was declared twice, which is not allowed. This results in a ``DeclarationError`` stating that the identifier was already declared.
 
-The tool that is being used for those tests is called ``isoltest`` and can be found under ``./test/tools/``. It is an interactive tool which allows
+The ``isoltest`` tool is used for these tests and you can find it under ``./test/tools/``. It is an interactive tool which allows
```

Comment:
What about specifying the full path already in the beginning?

Reply:
@chriseth That's a good catch!